http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/AzurePubsubHandler.scala

```diff
@@ -516,6 +523,150 @@ class AzurePubsubHandlerInterp[F[_]: Parallel](
       }
     } yield ()
 
+  private def monitorDeleteDisk(params: PollDeleteDiskParams)(implicit
+    ev: Ask[F, AppContext]
+  ): F[Unit] =
+    for {
+      ctx <- ev.ask
+      wsmApi <- buildWsmControlledResourceApiClient
+      getWsmJobResult = F.delay(wsmApi.getDeleteAzureDiskResult(params.workspaceId.value, params.jobId.value))
+
+      _ <- F.sleep(
```

Comment:
Do we need an initial delay here? My understanding is that the delete disk job is very quick

Reply:
Its only 5 seconds, I think its fine to have since its async and keeps the configs and format for polling monitoring the same class.